### PR TITLE
Added deborshikun.json to the list

### PR DESCRIPTION
**Pull request by deborshikun**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1526"></a>

### CODE REVIEW

File: public/data/deborshikun.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/deborshikun.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub!!!",
+      "url": "https://github.com/deborshikun",
+      "icon": "github"
+    }
+
```
Comment (CodingSpecies) [inline public/data/deborshikun.json:12]: Please can you remove line 12 @deborshikun . Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Comment by deborshikun** [comment]
Hey!!

Thanks for getting back to me!

Line 12 was just a blank line I am removing it!! Sorry for any
inconveniences!

Regards.

**Review by CodingSpecies — COMMENTED** [review]
Thank you @deborshikun for removing the line

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks for contributing! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

File: public/data/deborshikun.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Learning about Open Source Contributions",
+      "date": "July 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started Open Source learning",
+      "url": "https://www.eddiejaoude.io/"
```
Comment (EmmaDawsonDev) [inline public/data/deborshikun.json:20]: Did you mean to link to Eddie’s site here? Or would you like to change it to one of your own?

Comment (deborshikun) [reply public/data/deborshikun.json:20]: I don't have any websites to show to be honest so i thought let it be as it is. Should I change it to something else?

Comment (CodingSpecies) [reply public/data/deborshikun.json:20]: You could remove it if you like @deborshikun, as you don't need to have a url . Thank you.

Comment (deborshikun) [reply public/data/deborshikun.json:20]: Oh okay I got it!

File: public/data/deborshikun.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "title": "Started Learning about Open Source Contributions",
+      "date": "July 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started Open Source learning"
+    }
+  ]
+}
+
```
Comment (Panquesito7) [inline public/data/deborshikun.json:23]: There seems to be an extra line here. GitHub doesn't display the endline.
```suggestion
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

Comment (deborshikun) [reply public/data/deborshikun.json]: Yup I checked that too! seemed like I missed it on the vim. I have applied the necessary changes!

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**